Cargo.toml

```diff
@@ -15,24 +15,27 @@ dyn-clone = { version = "1.0", optional = true }
 fuel-asm = "0.1"
 fuel-merkle = "0.1"
 fuel-storage = "0.1"
-fuel-tx = "0.2"
+fuel-tx = { git = "https://github.com/fuellabs/fuel-tx.git", branch = "master"}
```

Comment:
This will be updated back to crates io once published.

Reply:
It's published, so should change back and test.

Reply:
Alright, it's out https://crates.io/crates/fuel-tx/0.3.0

Reply:
done